package ConstantsData;

import java.time.LocalDateTime;
import java.time.format.DateTimeFormatter;

public class ConstsntData {
	
	public static final String PROP_FILE_PATH="src/main/java/Global.properties";
	public static final String EXCEL_PATH="C:/Users/Vishnu/Documents/GSI_project/Maven_Project_userid.xlsx";
	public static final String FAILED_SCREEN_SHOT_PATH="target/FailedScreeenshot"+ LocalDateTime.now().format(DateTimeFormatter.ofPattern("yyyyMMdd"))
			                                                                                                        +"Test.png";
	
}
